platforms/cuda.jl

```diff
@@ -16,7 +16,7 @@ const augment = """
 
     function cuda_comparison_strategy(a::String, b::String, a_requested::Bool, b_requested::Bool)
         if a == "none" || b == "none"
-            return false
+            return a == b
         end
```

Comment:
cc: @maleadt any opposition to this change?

Reply:
Took me a while, but I think this is fine.